src/scripts/main.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  const promise = new Promise((resolve) => {
+    resolve(`
+      It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`);
+  });
```

Comment:
The promise in the `waitFor` function resolves immediately without waiting for the specified event to occur. You need to add an event listener to the element for the specified event and resolve the promise inside the event handler.